apps/demo-game/src/components/StoryElements.tsx

```
@@ -76,12 +83,7 @@ function StoryElements({ playerState, player }: Props) {
       </div>
 
       <div className="prose prose-img:max-w-xs prose-img:rounded mt-4 max-w-none">
-        <Markdown>
-          {(unseenStoryElements[0]?.type === 'GENERIC' &&
-            unseenStoryElements[0]?.content) ||
-            (unseenStoryElements[0]?.type === 'ROLE_BASED' &&
-              unseenStoryElements[0]?.contentRole?.[player.role])}
-        </Markdown>
+        <Markdown children={content} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)